code-challenges.js

```diff
@@ -7,16 +7,39 @@ var secretCodeWord1 = "lackadaisical"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
 
+//create a function that takes a string
+const code_Message = (str) => {
+//converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
+ let newString = str.replace("a", "4").replace("e", "3").replace("i", "1").replace("o", "0").replace("oo", "00")
```

Comment:
awesome job finding a function in js that does exactly the thing you need to do and also on chaining the method. Consider a solution that doesn't require you to loop through the string for each kind of replacement. Is there a data structure that you can leverage?